styles/main.css

```diff
@@ -259,12 +263,9 @@ body {
         grid-column: span 1;
     }
     
-    nav ul{
-        display: flex;
+    header nav ul{

```

Comment:
In order to override the previous `header nav ul` selector you need to have a selector that is AT LEAST as specific